ts/Extensions/DashboardPlugins/HighchartsComponent.ts

```diff
@@ -233,14 +232,23 @@ class HighchartsComponent extends Component<HighchartsComponent.ChartComponentEv
     }
 
     public render(): this {
-        this.emit({ type: 'beforeRender' });
-        super.render();
-        this.chart = this.initChart();
-        this.updateSeries();
-        this.sync.start();
-        this.emit({ type: 'afterRender' });
-        this.setupStoreUpdate();
+        const hcComponent = this;
 
+        hcComponent.emit({ type: 'beforeRender' });
+        super.render();
+        hcComponent.chart = hcComponent.initChart();
+        hcComponent.updateSeries();
+        hcComponent.sync.start();
+        hcComponent.emit({ type: 'afterRender' });
+        hcComponent.setupStoreUpdate();
+
+        addEvent(hcComponent.chart, 'update', function (e):void {
```

Comment:
It might be better to listen to `afterUpdate` instead, then special case are catched and you can access `this.options`.